scripts/workflows/release/rollback.cjs

```
@@ -22,159 +22,189 @@ function execGit(command, allowError = false) {
+        headers: {
+          Authorization: `token ${token}`,
+          "Content-Type": "application/json",
+        },
+      });
+
+      if (!response.ok) {
+        const text = await response.text();
+        lastError = new Error(`GitHub API error: ${response.status} ${response.statusText} - ${text}`);
+
+        if (response.status >= 500 && attempt < retries) {
+          await new Promise((resolve) => setTimeout(resolve, backoffMs));
```

ESLint

[error] 'setTimeout' is not defined.

(no-undef)